app/routes/account/billing-info/invoices.js

```diff
@@ -1,4 +1,34 @@
 import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
+import moment from 'moment';
 
 export default Route.extend({
+  async model() {
+    let filterOptions = [];
```

Comment:
this declaration of empty array is unnecessary ..

Reply:
Travis seems to fail with this message when I remove it.. .`ReferenceError: filterOptions is not defined`

Reply:
yes. you should be doing 

```js
const filterOptions = [
      {
        and: [
          {
            name : 'deleted-at',
            op   : 'eq',
            val  : null
          },
          {
            name : 'created-at',
            op   : 'ge',
            val  : moment().subtract(30, 'days').toISOString()
          }
        ]
      }
    ]
```